solutions/standard/main.tf

```diff
@@ -148,3 +148,47 @@ data "ibm_resource_instance" "existing_sm" {
   count      = var.existing_secrets_manager_crn == null ? 0 : 1
   identifier = var.existing_secrets_manager_crn
 }
+
+#######################################################################################################################
+# Secrets Manager Event Notifications Configuration
+#######################################################################################################################
+
+locals {
+  parsed_existing_en_instance_crn = var.existing_event_notification_instance_crn != null ? split(":", var.existing_kms_instance_crn) : []
+  existing_en_guid                = length(local.parsed_existing_en_instance_crn) > 0 ? local.parsed_existing_kms_instance_crn[7] : null
```

Comment:
you have referenced the `parsed_existing_kms_instance_crn` local instead of `parsed_existing_en_instance_crn`